io.github.lxi-tools.yaml

```diff
@@ -0,0 +1,133 @@
+---
+app-id: io.github.lxi-tools.lxi-gui
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: lxi-gui
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=home"
```

Comment:
I can't test it as it doesn't build yet, but if it operates on single files, and could use GtkNativeFileChooser to open and save files, this permission could be dropped.

Reply:
My app allows users via GtkFileChooser to save .lua scripts, .csv files, and .png image files and I would like users to be able to save those anywhere in their home - as everyone would.

Reply:
GtkFileNativeChooser should allow you to do all of that

Reply:
Yeah, but you see I don't really want that because that means that the user may be presented with a non-gtk4 native file chooser widget. I would much prefer to have the GTK4 file chooser working on the users home, hence the need for 'home' access.

Reply:
Sure, the next gnome release at least will have a gtk4 file chooser (thanks to xdg-desktop-portal-gnome) :)

Reply:
In flatpak the file chooser is part of the system and not the application surface area - that's why it's a legitimate security boundary. You should expect that a KDE or Qt file chooser (or anything else!) will show up if that's the base OS that the user has selected, the app author just gets or passes off a file. This reasoning is not compelling to allow the home permission.